openvdb_houdini/houdini/Utils.h

```diff
@@ -305,10 +305,45 @@ bool isLogForwarding(OP_OpTypeId);
 /// @}
 
 
+////////////////////////////////////////
+
+
+// Grid type lists, for use with openvdb::GridBase::apply()
+
+using ScalarGridTypes = openvdb::TypeList<
+    openvdb::BoolGrid,
+    openvdb::FloatGrid,
+    openvdb::DoubleGrid,
+    openvdb::Int32Grid,
+    openvdb::Int64Grid>;
+
+using NumericGridTypes = openvdb::TypeList<
+    openvdb::FloatGrid,
+    openvdb::DoubleGrid,
+    openvdb::Int32Grid,
+    openvdb::Int64Grid>;
+
+using RealGridTypes = openvdb::TypeList<
+    openvdb::FloatGrid,
+    openvdb::DoubleGrid>;
+
+using Vec3GridTypes = openvdb::TypeList<
+    openvdb::Vec3SGrid,
+    openvdb::Vec3DGrid,
+    openvdb::Vec3IGrid>;
+
+using PointGridTypes = openvdb::TypeList<
+    openvdb::tools::PointIndexGrid,
```

Comment:
I'm not sure there are many situations where an operation would apply to a PointDataGrid *and* a PointIndexGrid. I think I'd prefer "PointGridTypes" to be used just for a PointDataGrid. Also, not sure if it's relevant but PointIndexGrid isn't exposed in Houdini.

Reply:
`UT_VDB_POINTINDEX` is actually one of the supported grid types, though it's true that no SOPs output PointIndexGrids, and they're unlikely to be passed around by base class pointer.

Reply:
Yes, that's true.